apps/desktop/src/renderer/state/libraryStates.ts

```diff
@@ -103,8 +108,8 @@ export const sortedFilteredChapterListState = selector<Chapter[]>({
     const chapterFilterGroupNames = get(chapterFilterGroupNamesState);
     const chapterListVolOrder = get(chapterListVolOrderState);
     const chapterListChOrder = get(chapterListChOrderState);
-
     const uniqueChapters = new Map();
+    const toggleReadChapters = get(ReadChaptersState);
```

Comment:
`toggleReadChapters` doesn't make sense, this is a boolean. Maybe also change `ReadChaptersState` to something like `hideUnreadChaptersState` and adjust accordingly?